lib/tasks.js

```diff
@@ -39,6 +39,37 @@ var removeTimer = function(timeoutId, callback, Zone){
 	return callback();
 };
 
+var getGlobalEventHandlersNames = function getGlobalEventHandlersNames() {
+	return Object.getOwnPropertyNames(HTMLElement.prototype).filter(
```

Comment:
[Object.getOwnPropertyName](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames) is ES5, that's why I didn't make the call conditional.... We'll have to do the same for `Object.defineProperty` and `Object.getOwnPropertyDescriptor`.